.github/ISSUE_TEMPLATE/missing_documentation.md

```diff
@@ -11,6 +11,10 @@ assignees: ''
 
 <!-- describe your problem -->
 
+## Proposal
```

Comment:
I moved it back down because I perceived it as too passive aggressive. Also it was weird reading issues that started with a checklist instead of what the issue actually is about.